.github/workflows/push-on-branch.yml

```diff
@@ -0,0 +1,24 @@
+name: Build on Pushes to Branches
+
+on:
+  push:
+    branches: [ main, develop ]
+
+env:
+  TR_PORT: 3008
+  TR_PATH_PREFIX: fit-and-competent
+
```

Comment:
You probably don't need these as you've defined them in the 'relaxed' style [here](https://github.com/Scottish-Natural-Heritage/fit-and-competent/pull/1/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R5) and [here](https://github.com/Scottish-Natural-Heritage/fit-and-competent/pull/1/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R8).

Reply:
Resolved in acd360fa8c05a23efca40928c563bde884faab98.